taichi/transforms/fix_block_parents.cpp

```diff
@@ -46,6 +46,7 @@ class FixBlockParents : public IRVisitor {
   }
 
   void visit(OffloadedStmt *stmt) override {
+    stmt->parent = current_block; // hot fix
```

Comment:
Some tests are failing without this hotfix. This pass should be fixing `Block`'s parents but not `OffloadedStmt`'s. I think we can fix this systematically in the future.